src/glib/base/ut.cpp

```diff
@@ -129,12 +129,45 @@ void TStdErrNotify::OnStatus(const TStr& MsgStr){
 // Log-Notify
 void TLogNotify::OnStatus(const TStr& MsgStr) {
 	TTm NowTm = TTm::GetCurLocTm();
-	Notify->OnStatus(TStr::Fmt("[%s %s] %s", 
-		NowTm.GetYMDDashStr().CStr(), 
-		NowTm.GetHMSTColonDotStr(true, false).CStr(), 
+	Notify->OnStatus(TStr::Fmt("[%s %s] %s",
+		NowTm.GetYMDDashStr().CStr(),
+		NowTm.GetHMSTColonDotStr(true, false).CStr(),
 		MsgStr.CStr()));
 }
 
+//////////////////////////////////////
+// Color-Notify
+// CODE	   COLOR
+// [0;31m	 Red
+// [1;31m	 Bold Red
+// [0;32m	 Green
+// [1;32m	 Bold Green
+// [0;33m	 Yellow
+// [01;33m	Bold Yellow
+// [0;34m	 Blue
+// [1;34m	 Bold Blue
+// [0;35m	 Magenta
+// [1;35m	 Bold Magenta
+// [0;36m	 Cyan
+// [1;36m	 Bold Cyan
+// [0m	   Reset
+
+void TColorNotify::OnStatus(const TStr& MsgStr) {
```

Comment:
Requires consistent number of indentation spaces. The rest of the code has 4 spaces.